src/common/NodeSpecs/lighthouse/lighthouse-v1.0.0.json

```diff
@@ -115,6 +117,33 @@
       "infoDescription": "NiceNode requires http connections",
       "documentation": "https://lighthouse-book.sigmaprime.io/api-bn.html#starting-the-server"
     },
+    "quicPort": {
+      "displayName": "QUIC Connections",
+      "uiControl": {
+        "type": "select/single",
+        "controlTranslations": [
+          {
+            "value": "Enabled"
+          },
+          {
+            "value": "Disabled",
+            "config": "--disable-quic"
+          }
+        ]
+      },
+      "defaultValue": "Enabled",
+      "documentation": "https://lighthouse-blog.sigmaprime.io/Quic.html"
+    },
+    "quicPortUdp": {
+      "displayName": "QUIC port (UDP)",
+      "cliConfigPrefix": ["--quic-port ", "--quic-port6 "],
```

Comment:
Did you happen test with both flags? I can't say I know much about ipv6. If it works with both and this covers both cases, then lgtm!

Reply:
I tried running with both flags on, but didn't see any issues from what I can see. I don't know much about ipv6 either.. 😅